src/support/bytemuck.rs

```diff
@@ -0,0 +1,79 @@
+//! Support for the [`bytemuck`](https://crates.io/crates/bytemuck) crate.
+#![cfg(feature = "bytemuck")]
+#![cfg_attr(docsrs, doc(cfg(feature = "bytemuck")))]
+
+use bytemuck::{Pod, Zeroable};
+use ruint::Uint;
+
+// Implement Zeroable for all `Uint` types.
+unsafe impl<const BITS: usize, const LIMBS: usize> Zeroable for Uint<{ BITS }, { LIMBS }> {}
+
+// Implement the `Pod` trait for `Uint` types with a size that is a multiple of 64, up to 1024.
+macro_rules! impl_pod {
+    ($(($bits:expr, $limbs:expr)),+ $(,)?) => {
+        $(
+            unsafe impl Pod for Uint<{$bits}, $limbs> {}
+        )+
+    };
+}
+
+impl_pod! {
```

Comment:
is it possible to replace this with a `generic_const_expr` usage when `cfg(feature = "nightly")` ?

Reply:
i don't consider this a blocker